src/components/SecondaryPanes/Workers.css

```diff
@@ -0,0 +1,13 @@
+.workers-list * {
+  -moz-user-select: none;
+  user-select: none;
+}

```

Comment:
We use autoprefix so you don't need the moz prefix

Reply:
I looked how breakpoints are styled so I'll remove it there too